packages/suite/src/views/settings/device/Homescreen.tsx

```diff
@@ -103,20 +102,22 @@ export const Homescreen = ({ isDeviceLocked }: HomescreenProps) => {
                 ref={anchorRef}
                 shouldHighlight={shouldHighlight}
             >
-                {isModel1 ? (
+                {[DeviceModel.T1, DeviceModel.TR].includes(deviceModel) && (
```

Comment:
I would have a config somewhere for touch/buttons, screen types, screen rotation to avoid using `.includes()`

Reply:
It is mentioned here https://www.notion.so/satoshilabs/Model-R-9e5a40f57fc047e3bdd03b4f5404ca4f#29471d82bae4478fb5842cb0107ec8d1 and in the PR description as not part of this PR

Reply:
Good! Sure, I didn't mean it has to be implemented here, I was just wondering why it's not in `capabilities` already.